aten/src/ATen/native/MaxPooling.h

```diff
@@ -3,9 +3,68 @@
 #include <ATen/core/Tensor.h>
 #include <ATen/Parallel.h>
 #include <ATen/native/DispatchStub.h>
+#include <ATen/native/Pool.h>
 
 namespace at::native {
 
+namespace {
```

Comment:
Why this namespace in the header?

Reply:
I copy-pasted it from the other file, but I can also remove it.